README.md

```diff
@@ -1,4 +1,42 @@
 # misc-files
-Repository for miscellaneous files frequently overwritten by upstreams
 
 [![Periodic updates](https://github.com/AOSC-Dev/misc-files/actions/workflows/periodic-update.yml/badge.svg)](https://github.com/AOSC-Dev/misc-files/actions/workflows/periodic-update.yml)
+
+Repository for miscellaneous files frequently overwritten by upstreams.
+
+This repository is driven by the tracking list file `files.list` and utilizes GitHub Actions to keep files updated. 
+
+## CI/CD
+
+The main workflow `periodic-update` is triggered under the following conditions:
+
+- Update `files.list` on branch `master`
+- Cron `"0 0 * * *"`
+- Manually triggered by maintainers
+
+As of update this repository is tracking and being referenced by the following packages:
```

Comment:
Move this paragraph up and change this sentence:

"Files in this repository are being used by the following packages:"

You might also want to note the list of files (or even a glob) corresponding to each package.